charts/drupal/templates/varnish-configmap-vcl.yaml

```diff
@@ -15,19 +15,7 @@ data:
     backend prod1_http1 {
       .host = "{{ .Release.Name }}-drupal";
       .port = "80";
-      .max_connections = 300;
-
-      .probe = {
-        .url       = "/robots.txt"; 
-        .interval  = 5s; # check the health of each backend every n seconds
```

Comment:
Let's remove those doc comments. Variable names are self explanatory on the most part and there's also a link to VCL file at the top. It will make the yaml file look slightly cleaner, less busy and more readable.

Reply:
Makes sense! Done (in `charts/drupal/values.yaml`)